File: day02/Nilesh_18311/Product.java
package com.sonata;

public class Product {
	int prdId;
	String prdName;
	float prdPrice;
	double totalPrice;
	
	public Product(int prdId,String Name,float price)
	{
		this.prdId=prdId;
		this.prdName=Name;
		this.prdPrice= price;
		totalPrice=price+(price*.18);
	}
	public void display()
	{
		System.out.println(prdName+" "+ totalPrice);
	}
	
	public static void main(String[] args) {
	
		Product p1=new Product(001,"Skoda",10);
		p1.display();
		
	}

}

File: JS/Day1Assign/Product.js
var Product = /** @class */ (function () {
    function Product(id, name, price) {
        this.prdId = id;
        this.prdNamme = name;
        this.prdPrice = price;
    }
    Product.prototype.additionalGst = function () {
        var totalCost = this.prdPrice + this.prdPrice * 0.18;
        return totalCost;
    };
    Product.prototype.display = function () {
        console.log("Product Id: " + this.prdId + ", Product: " + this.prdNamme + ", Product Price with GST :" + this.additionalGst());
    };
    return Product;
}());
var prd1 = new Product(201, "LapTop", 25000);
prd1.display();
var prd2 = new Product(202, "iPhone", 50000);
prd2.display();
